src/cmd_parser.c

```diff
@@ -213,6 +213,11 @@ static GOptionEntry cmd_entries[] =
       "Read the list of packages from old metadata directory and re-use it.  This "
       "option is only useful with --update (complements --pkglist and friends).",
       NULL },
+    { "delayed-dump", 0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE,
```

Comment:
Is there any benefit to having this as a hidden option? Maybe for debugging? But personally I wouldn't add it here at all.

Reply:
I used it several times for debugging, yes.  There shouldn't be cons in having the option, so I'd prefer to keeping it?  Just please re-consider, and I'll drop if you really want.

Reply:
ACK, dropped the command line option then, thanks for checking. I kept the "internal" delayed_dump option just to not overcomplicate the commit changes if that is OK?

Reply:
Great, thanks!